ui-components/src/global/markdown.scss

```diff
@@ -25,9 +25,15 @@
   ul:not(:last-child) {
     @apply mb-4;
   }
-
   ul {
-    list-style: disc;
+    list-style: lower-alpha;
```

Comment:
Since this is a global style file, we are using this disc styling in markdown files elsewhere, so this would cause a regression for the places we are expecting the disc method. We will need to pass this style into the markdown from the terms page itself to stylize that.